py3status/modules/spotify.py

```diff
@@ -9,6 +9,10 @@
         (default 'Spotify not running')
     format_stopped: define output if spotify is not playing
         (default 'Spotify stopped')
+    sanitize_titles: whether to remove meta data from album/track title
+        (default True)
+    sanitize_words: which meta data to remove separated by |
+        (default 'stereo|mono|remaster|edit|bonus|extended|demo|explicit|version|feat')
```

Comment:
yes https://github.com/ultrabug/py3status/blob/master/tests/test_module_doc.py#L37 is the place to do it.

The docsting parsing is a bit primitive in the tests.